WooCommerce/StoreWidgets/Homescreen/StoreInfoView.swift

```diff
@@ -97,15 +97,15 @@ private struct StatsCard: View {
 
             // Orders & Conversion
             HStack {
-                VStack(alignment: .leading, spacing: StoreInfoView.Layout.cardSpacing) {
-                    Text(StoreInfoView.Localization.orders)
-                        .statTitleStyle()
-
-                    Text(entryData.orders)
-                        .statValueStyle()
+                Link(destination: URL(string: ordersDeepLink)!) {
```

Comment:
Do you think we can avoid the `!`?

We have a `asURL()` in `WooConstant` that adds a tiny extra layer of safety 😇

Reply:
Thanks for the tip!